BuildingBlocks/src/BuildingBlocks.API/Extensions/HostExtensions.cs

```diff
@@ -69,7 +69,7 @@ public static class HostExtensions
     }
 }
 
-public interface IDbSeeder<in T> where T : DbContext
+public interface IDbSeeder<in T>
```

Comment:
Why did you delete this constraint?

Reply:
No idea, I don't even remember touching the file or seeing it in the changes. My bad & reverted.